tests/x11/gnomecase/application_starts_on_login.pm

```diff
@@ -26,7 +26,10 @@ sub tweak_startupapp_menu {
     }
     else {
         # tweak-tool entry is not in gnome-control-center of SLE15;
-        x11_start_program 'gnome-tweak-tool';
+	if (is_tumbleweed) {
+	       	x11_start_program 'gnome-tweaks';
+	}
+	else x11_start_program 'gnome-tweak-tool';
```

Comment:
Please use brackets for the else:

```suggestion
	else {
		x11_start_program 'gnome-tweak-tool';
	}
```

... and run `./tools/tidy` to fix your code indentation.